### PR TITLE
Littlechange

### PR DESCRIPTION
**Pull request by tanmoydas9166**
Please add me to the list  @whimboo @tirkarthi

### CODE REVIEW

File: meet.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from selenium import
```
Comment (whimboo) [inline meet.py:1]: Why did you add this file? It shouldn't be part of this PR.

File: setup.py [hunk 2]
```diff
@@ -18,8 +18,8 @@ def read(*parts):
 
 
 def get_version():
-    return re.findall("__version__ = '([\d\.]+)'",
-                      read('mozdownload', 'cli.py'), re.M)[0]
+	p = re.compile(r"__version__ = '(\d.+)'")
```
Comment (whimboo) [inline setup.py:21]: What was basically requested is to get rid of `\d` by replacing it with `0-9`, and not changing all the code around the regex. Please get this reverted/updated.

**Review by whimboo — CHANGES_REQUESTED** [review]
Please see my inline comments for what's remaining to do. Also please find a better wording for the commit message, which would help to later identify the changes as being done on this PR. Thanks.

**Comment by whimboo** [comment]
Fixes issue #609.

File: setup.py (hunk 2 above)
Comment (tanmoydas9166) [reply setup.py:21]: **I was thinking of converting** 
    return re.findall("__version__ = '([\d\.]+)'",
                      read('mozdownload', 'cli.py'), re.M)[0]
**to a raw string**
    return re.findall(r"__version__ = '([\d\.]+)'",
                      read('mozdownload', 'cli.py'), re.M)[0]
Will that work?
Help appreciated.

Comment (whimboo) [reply setup.py:21]: That should work too. Make sure to be able to reproduce the deprecation warning first with Python3. Then it makes it easier for you to verify the changes.

**Comment by whimboo** [comment]
@tanmoydas9166 do you have an update for us? Thanks.

**Comment by whimboo** [comment]
Closing due to inactivity.
